.babelrc

```diff
@@ -1,5 +1,8 @@
 // .babelrc
 {
   "presets": [ "es2015-rollup" ],
-  "plugins": ["./babel-strip-class-call-check"]
+  "plugins": [
+    "./babel-strip-class-call-check",
+    "transform-proto-to-assign"
```

Comment:
```js
class Foo {
  static soemthing() {
  }
}
```

Is transpiled to use `__proto__` in some circumstances so I included it "just in case" I suppose.  I can remove it and double check transpiled output if you'd like?